.jenkins/changeMaster.groovy

```diff
@@ -0,0 +1,271 @@
+// SPDX-FileCopyrightText: 2022-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
This is the first draft of a job that could run daily/weekly to test if the latest master SHA is a good candidate to override the locally built image.